src/wp-includes/load.php

```diff
@@ -164,8 +164,7 @@ function wp_check_php_mysql_versions() {
 
 	if ( ! function_exists( 'mysqli_connect' ) && ! function_exists( 'mysql_connect' )
 		// This runs before default constants are defined, so we can't assume WP_CONTENT_DIR is set yet.
-		&& ( defined( 'WP_CONTENT_DIR' ) && ! file_exists( WP_CONTENT_DIR . '/db.php' )
-			|| ! file_exists( ABSPATH . 'wp-content/db.php' ) )
+		&& ! ( defined( 'WP_CONTENT_DIR' ) ? file_exists( WP_CONTENT_DIR . '/db.php' ) : file_exists( ABSPATH . 'wp-content/db.php' ) )
```

Comment:
I wonder if putting the `!` before each `file_exists()` would ensure those reading this code in future don't miss the `!` at the beginning of this condition. What do you think?

Reply:
In one sense that may be a good point. But then we'd be using the `!` twice & may sound repeated & not reusable?
I don't really have a strong opinion here.
May be if there's a similar code example in the codebase we can follow that for consistency. But, I haven't came across any similar example yet.

I think it'd be great if we can find a similar code example to follow.

Reply:
Great idea! I think the original code works as an example of using `!` before each call for clarity, but to find another example, [see here](https://github.com/costdev/wordpress-develop/blob/b66bbf56c678e2559219e6d9af7e4248800b9b2a/src/wp-includes/class-wp-theme.php#L365-L369), where three `file_exists()` checks are each prepended with `!`.

Reply:
Thanks @costdev for sharing the example but I wouldn't call it exactly simillar for one that's `not ternary` and another using `!` once with the 2 `&&` shown in your example won't be the same as it currently is. 

I'd call [this example I found](https://github.com/WordPress/wordpress-develop/blob/c47f438376fd26a15a13f01667aa99645a09d2b4/src/wp-admin/authorize-application.php#L68-L70) similar to what we're looking for.

there's 'may/may not' opposite be found in the codebase but I think, we can follow the example I shared as it won't be repetitive. 
But, that's not a strong opinion by me. 😊